packages/core/core/src/BundleGraph.js

```diff
@@ -574,6 +574,8 @@ export default class BundleGraph {
     return hashHex;
   }
 
+  // ? Should this be removed? Or at least renamed to signify that it incorporates content of child bundles,
+  //  - even though that may not affect the content of the output bundle
```

Comment:
Thoughts?

Reply:
Minimum renamed. If it no longer serves a purpose, I vote remove.

Reply:
I decided to remove `getHash` from the public bundle API as plugins should only be using hash references. Since right now only `PackagerRunner` is using it, I changed it to only consider inline bundles.